content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -29,3 +29,26 @@ cluster must satisfy the following requirements:
   `app` label is used to add contextual information in distributed
   tracing. The `app` and `version` labels are also used to add contextual information
   in the metric telemetry collected by Istio.
+
+* _**`NET_ADMIN` capability**:_ The pods must have the `NET_ADMIN` capability allowed. It is required for the init
+  containers of the Istio sidecar proxies. To check which capabilities are allowed for your pods, check if their
+  [Service Account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) can use a
+  [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) that allows the `NET_ADMIN`
+  capability. (Note that if the Service Account is not specified in the Deployment of your pods, they run as the
+    `default` Service Account in the namespace they are deployed.)
+  To check which capabilities are allowed for the Service Account of your pods, run the following command:
```

Comment:
K8S does not have a doc that explains this. Should I add a task "Check that your pods have NET_ADMIN permission", under Setup?

Reply:
@rcaballeromx So what is your opinion - should I add a task "Check that your pods have NET_ADMIN permission", under Setup?

Reply:
@rcaballeromx I have split the item, extracted the most of it into Operations Guide -> Required Pod Capabilities. Added mentioning "unless you use Istio CNI Plugin".

I change your suggested sentence to be: "To verify whether your pods have the right permissions, ensure the [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) of your pods can use
a [Pod security policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) that allows the `NET_ADMIN` capability." I used "... policy that allows" instead of your suggested "... policy allowing".

Could you please review it again?